common-services-api/pig/pom.xml

```diff
@@ -53,7 +53,8 @@
         <configuration>
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-            <Export-Package>org.pentaho.bigdata.api.pig,org.pentaho.bigdata.api.pig.impl</Export-Package>
+            <Bundle-Version>${pentaho-osgi-bundles.version}</Bundle-Version>
+            <Export-Package>org.pentaho.bigdata.api.pig.*;version=${pentaho-osgi-bundles.version}</Export-Package>
```

Comment:
This is properly split into interface and implementation. We only need to export the interface package.